datajunction-server/pyproject.toml

```diff
@@ -48,6 +47,7 @@ dependencies = [
     "types-cachetools>=5.3.0.6",
     "strawberry-graphql>=0.204.0",
     "fastapi-cache2>=0.2.1",
+    "psycopg2>=2.9.9",
```

Comment:
Could we use `psycopg` here? That would get us v3 (https://pypi.org/project/psycopg/#history)

Reply:
Good point! Let me try rerunning the tests with `psycopg` v3